Cargo.toml

```diff
@@ -14,6 +14,9 @@ exclude = [
     ".travis.yml", ".gitignore", "test-data/**"
 ]
 
+[package.metadata.docs.rs]
+rustdoc-args = ["--cfg", "docsrs"]
+
```

Comment:
i don't think this is needed, `#[cfg(docsrs)]` is automatically applied according to https://docs.rs/about/builds#detecting-docsrs.

Reply:
I didn't know this. I fixed this.